bench_test.go

```diff
@@ -0,0 +1,38 @@
+package triangle
+
+import (
+	"bytes"
+	"image"
+	_ "image/png"
+	"os"
+	"testing"
+)
+
+func BenchmarkDraw(b *testing.B) {
+	buf, err := os.ReadFile("./output//sample_0.png")
```

Comment:
Because Triangle works with older Go versions could you please use `ioutil.ReadFile` instead of `os.ReadFile` which has been included in the latest Go v1.16. Otherwise everything is just fine and I also noticed a quite significant performance boost by using your PR.

Reply:
Done!